CONTRIBUTORS.md

```diff
@@ -0,0 +1,53 @@
+# Contributors to RGBDS
+
+## Original author
+
+- Carsten Elton Sørensen &lt;csoren@gmail.com&gt;
+
+## Main contributors
+
+- Justin Lloyd &lt;jlloyd@imf.la&gt;
+
+- Vegard Nossum &lt;vegard.nossum@gmail.com&gt;
+
+- Anthony J. Bentley &lt;anthony@anjbe.name&gt;
+
+- stag019 &lt;stag019@gmail.com&gt;
+
+- Antonio Niño Díaz &lt;antonio_nd@outlook.com&gt;
+
+- Eldred "ISSOtm" Habert &lt;eldredhabert0@gmail.com&gt;
+
+- Sylvie "Rangi" Oukaour &lt;http://github.com/Rangi42&gt;
+
+## Other contributors
+
+- Ben Hetherington &lt;dev@ben-h.uk&gt;
+
+- Björn Höhrmann &lt;bjoern@hoehrmann.de&gt;
+
+- Christophe Staïesse &lt;chastai@skynet.be&gt;
+
+- David Brotz &lt;dbrotz007@gmail.com&gt;
+
+- Jakub Kądziołka &lt;kuba@kadziolka.net&gt;
```

Comment:
I know @meithecatte has changed names. Should we follow the rename?

Reply:
Yes, thanks!